scripts/merge-buddyup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# syntax:
+# merge-buddyup.sh locale-dir/
+
+function usage() {
+    echo "syntax:"
+    echo "merge-buddyup.sh locale-dir/"
+    exit 1
+}
+
+# check if file and dir are there
+if [[ ($# -ne 1) || (! -d "$1") ]]; then
+    usage
+fi
+
+POTFILE=`find $1 -type f -name "buddyup.pot"`
```

Comment:
I was mimicking the structure of `compile-mo.sh` as much as possible to reduce differenceness in the code-base. If you don't want that, that's fine. Then we should add some code to verify that `buddyup.pot` exists in the first place and some other things.